src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -115,8 +115,8 @@ public function mkdir($dirs, $mode = 0777)
     public function exists($files)
     {
         foreach ($this->toIterator($files) as $file) {
-            if ('\\' === DIRECTORY_SEPARATOR && strlen($file) > 258) {
-                throw new IOException('Could not check if file exist because path length exceeds 258 characters.', 0, null, $file);
+            if (strlen($file) > PHP_MAXPATHLEN) {
```

Comment:
PHP_MAXPATHLEN == 260 on Windows, so there is a 2 offset that needs consideration.

Reply:
This is really.

Reply:
So PHP on Windows exposes the wrong max path len ? Has it ever been reported as a bug ?

Reply:
@stof This is not a bug. I checked the PHP source code and they thought 256 was safe.  Limited to 258 should be compatible with more windows operating system.

- Link: https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx
- Link: https://github.com/php/php-src/blob/master/main/php.h  [line 324]